libs/dh/admin/feature-user-management/src/lib/drawer/dh-user-drawer.component.html

```diff
@@ -77,10 +72,6 @@ <h2>{{ selectedUser()?.firstName + " " + selectedUser()?.lastName }}</h2>
     }
   </watt-drawer>
 
-  @if (isEditUserModalVisible) {
-    <dh-edit-user-modal [user]="selectedUser()!" (closed)="modalOnClose()" />
-  }
```

Comment:
praise: nice change. Opening the modal dynamically works a lot better than keeping an extra property to avoid initializing the component.